org.standardnotes.standardnotes.yml

```diff
@@ -0,0 +1,163 @@
+app-id: org.standardnotes.standardnotes
+branch: stable
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '20.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+# Electron doesn't use a traditional locale format so separate-locales is useless.
+separate-locales: false
+command: start-standardnotes
+finish-args:
+  - '--device=dri'
+  - '--filesystem=xdg-download'
+  - '--filesystem=xdg-documents'
+  - '--filesystem=xdg-run/keyring'
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=x11'
+  - '--talk-name=org.freedesktop.Notifications'
+  - '--talk-name=org.freedesktop.secrets'
+modules:
+  - name: yarn
+    buildsystem: simple
+    build-commands:
+      - 'cp -a * /app'
+    # Only used for building, so clean it up afterwards.
+    cleanup:
+      - '*'
+    sources:
+      - type: archive
+        url: https://github.com/yarnpkg/yarn/releases/download/v1.22.10/yarn-v1.22.10.tar.gz
+        sha256: 7e433d4a77e2c79e6a7ae4866782608a8e8bcad3ec6783580577c59538381a6e
+
+  - name: python2
+    buildsystem: autotools
+    # Only used for building, so clean it up afterwards.
+    cleanup:
+      - '*'
+    sources:
+      - type: archive
+        url: https://www.python.org/ftp/python/2.7.18/Python-2.7.18.tar.xz
+        sha256: b62c0e7937551d0cc02b8fd5cb0f544f9405bafc9a54d3808ed4594812edef43
+
+  - name: standardnotes
+    buildsystem: simple
+    build-options:
+      # Add the node bin directory.
+      append-path: '/usr/lib/sdk/node14/bin:/app/yarn/bin:/run/build/standardnotes/flatpak-node/chromedriver/'
+      env:
+        # Don't add ELECTRON_CACHE
+        XDG_CACHE_HOME: /run/build/standardnotes/flatpak-node/cache
+        # ELECTRON_CACHE: '/run/build/standardnotes/flatpak-node/electron-cache/'
+        CHROMEDRIVER_SKIP_DOWNLOAD: 'true'
+        JOBS: max
+        npm_config_nodedir: /run/build/standardnotes/flatpak-node/cache/node-gyp/9.4.2
+    build-commands:
+      # Retire this at some point; (1) create a new task by default and (2) use Yarn instead"
+      - "sed -i '/\\\"pack\\\"/c\\    \\\"pack:linux\\\": \\\"npm run
+        build:linux && electron-builder --linux\\\",' package.json"
```

Comment:
Adding task is not needed.
You can do 
```
- . flatpak-node/electron-builder-arch-args.sh && 
./node_modules/.bin/electron-builder --linux $ELECTRON_BUILDER_ARCH_ARGS --dir
```
Also need run `npm(yarn) run bundle` manually.